docs/source/gbd2017_models/causes/neoplasms/index.rst

```diff
@@ -0,0 +1,37 @@
+.. _2017_cause_neoplasms:
+
+==============================
+Neoplasm (Cancer) Cause Models
```

Comment:
is this page eventually going to have some of the additional headings/structure of the other pages? (experts v gbd v vivarium?)

Reply:
ohh i was misinterpreting; pls disregard